expression/constant.go

```diff
@@ -448,3 +449,13 @@ func (c *Constant) Vectorized() bool {
 	}
 	return true
 }
+
+// SupportReverseEval checks whether the builtinFunc support reverse evaluation.
+func (c *Constant) SupportReverseEval() bool {
+	return true
+}
+
+// ReverseEval evaluates the only one column value with given function result.
+func (c *Constant) ReverseEval(sc *stmtctx.StatementContext, res types.Datum, rType types.RoundingType) (val types.Datum, err error) {
+	return types.Datum{}, errors.Errorf("Constant.ReverseEval() should never be called, please contact the TiDB team for help")
```

Comment:
Can we return `c.Value` here, which seems more reasonable?

Reply:
OK, I'll change that.